DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/Common/DevExtreme CLI/20 DevExtreme Application/10 Create a New Application.md

    See the following topic to review both layouts: [Layouts](/concepts/50%20React%20Components/50%20Application%20Template/10%20Layouts.md '/Documentation/Guide/React_Components/Application_Template/#Layouts').

* `--empty`     
Specifies whether to skip sample data generation. Available values:
    * `false` (default)
    * `true` 

* `--template`

> Specifies whether to create an application template with TypeScript support. Available values:

Reviewer: "Whether" is good for Boolean options. But here, I'd just say "specifies the language". 

```suggestion
Specifies the language you can use in the application. Available values:
```

    * `javascript`
    * `typescript`

* `--app-type`

> Specifies whether to create an application with [Next.js](https://nextjs.org/) or [Vite](https://vite.dev/). Available values:

Reviewer: ```suggestion
Specifies the framework used in the application: [Next.js](https://nextjs.org/) or [Vite](https://vite.dev/). Available values:
```

    * `nextjs`
    * `vite`

* `--transpiler`

> Specifies whether to use [Babel](https://babeljs.io/) or [SWC](https://swc.rs/) as a transpiler. Used only when `--app-type` is `vite`. Available values:

Reviewer: ```suggestion
Specifies the transpiler: [Babel](https://babeljs.io/) or [SWC](https://swc.rs/). Used only when `--app-type` is `vite`. Available values:
```
